kastenwesen.py

```diff
@@ -422,14 +429,15 @@ def stop(self):
     def start(self):
         if self.is_running():
             raise Exception('container is already running')
-        container_id_file = "{}.running_container_id".format(self.name)
+        name_without_ns = self.name_without_namespace()
+        container_id_file = "{}.running_container_id".format(name_without_ns)
         # move container id file out of the way if it exists - otherwise docker complains at startup
         try:
             os.rename(container_id_file, container_id_file + "_previous")
         except OSError:
             pass
         # names cannot be reused :( so we need to generate a new one each time
-        new_name = self.name + datetime.datetime.now().strftime("-%Y-%m-%d_%H_%M_%S")
+        new_name = name_without_ns + datetime.datetime.now().strftime("-%Y-%m-%d_%H_%M_%S")
```

Comment:
Why are the container (instance) names supposed to be without namespace? `namespace1/webserver` and `namespace2/webserver` would then have the same instance names `webserver-$date`.